profiles/pythonhubdev.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/60695664/194086997-2673cbdd-f7ad-4ba3-8980-ec2500de3fe8.png",
+  "issueId": 1467,
+  "name": "Vetrichelvan",
+  "username": "pythonhubdev"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```